--- openembedded/packages/bluez/bluez4_4.30.bb.orig	2009-02-09 15:18:40.000000000 +0100
+++ openembedded/packages/bluez/bluez4_4.30.bb	2009-02-09 15:20:07.000000000 +0100
@@ -8,9 +8,7 @@
 
 SRC_URI = "\
   http://www.kernel.org/pub/linux/bluetooth/bluez-${PV}.tar.gz \
-  file://fix-dfutool-usb-declaration-mismatch.patch;patch=1 \
   file://sbc-thumb.patch;patch=1 \
-  file://hid2hci_usb_init.patch;patch=1 \
 "    
 S = "${WORKDIR}/bluez-${PV}"
 
