--- openembedded/packages/linux/linux-openmoko-devel_git.bb	2009-02-24 23:16:15.000000000 +0100
+++ openembedded/packages/linux/linux-openmoko-devel_git.bb.new	2009-02-24 23:15:04.000000000 +0100
@@ -8,9 +8,9 @@
 KERNEL_RELEASE = "2.6.28"
 KERNEL_VERSION = "${KERNEL_RELEASE}"
 
-OEV = "oe1"
-PV = "${KERNEL_RELEASE}-${OEV}+gitr${SRCREV}"
-PR = "r2"
+OMV = "andy-tracking"
+PV = "${KERNEL_RELEASE}-${OMV}+gitr${SRCREV}"
+PR = "r3"
 
 SRC_URI = "\
   git://git.openmoko.org/git/kernel.git;protocol=git;branch=andy-tracking \
