--- openembedded/conf/checksums.ini~    2009-02-07 16:39:49.000000000 +0100
+++ openembedded/conf/checksums.ini     2009-02-08 00:17:32.000000000 +0100
@@ -1,3 +1,7 @@
+[http://effbot.org/media/downloads/elementtree-1.2.7-20070827-preview.zip]
+md5=30e2fe5edd143f347e03a8baf5d60f8a
+sha256=5071431068c58c1f56dcc8fff37f8a213351f3b45c012d3adb640ec9418053ad
+
 [http://www.directfb.org/downloads/Extras/++DFB-0.9.25.tar.gz]
 md5=c6a2705f6210d8ede50a947b375f1c0b
 sha256=b3dbf01563dd60d63cff543ea8cd0c885381e78ecbbab8ff176e46df3f198a88
