--- openembedded/conf/distro/include/sane-srcrevs.inc.orig	2009-02-24 23:19:53.000000000 +0100
+++ openembedded/conf/distro/include/sane-srcrevs.inc	2009-02-24 23:21:51.000000000 +0100
@@ -97,6 +97,7 @@
 SRCREV_pn-linux-openmoko-2.6.24 ?= "fb42ce6724576fc173faf8abfb04aa2c36d213b7"
 SRCREV_pn-linux-openmoko-2.6.28 ?= "34240a1c06ae36180dee695aa25bbae869b2aa26"
 SRCREV_pn-linux-openmoko-devel ?= "34240a1c06ae36180dee695aa25bbae869b2aa26"
+SRCREV_pn-linux-openmoko-stable ?= "0e5fe639e234cdeb11d8441f19c5b3109a8b6a17"
 SRCREV_pn-linux-eten ?= "ab2d414f4f13816af0ec0401b608133ca946624a"
 SRCREV_pn-llvm-gcc4 ?= "374"
 SRCREV_pn-llvm-gcc4-cross ?= "374"
@@ -202,7 +203,7 @@
 SRCREV_pn-qemu-native ?= "${QEMUSRCREV}"
 SRCREV_pn-qemu-sdk ?= "${QEMUSRCREV}"
 SRCREV_pn-qemu ?= "${QEMUSRCREV}"
-SRCREV_pn-qi ?= "ed52486b43dca91891305b9e832c8366a2742798"
+SRCREV_pn-qi ?= "1fc83905fe99da92ff18e6ba671ee339477b1b9e"
 SRCREV_pn-remoko ?= "121"
 SRCREV_pn-remoko-server ?= "121"
 SRCREV_pn-s3c24xx-gpio ?= "4130"
